entity/templates/src/main/resources/config/cql/_added_entity.cql

```diff
@@ -3,7 +3,7 @@ CREATE TABLE IF NOT EXISTS <%= entityInstance %> (
     id uuid,<% for (fieldId in fields) { %><% if (fields[fieldId].fieldType == 'UUID') { %>
     <%= fields[fieldId].fieldName %> uuid,<% } else if (fields[fieldId].fieldType == 'TimeUUID') { %>
     <%= fields[fieldId].fieldName %> timeuuid,<% } else if (fields[fieldId].fieldType == 'String') { %>
-    <%= fields[fieldId].fieldName %> text,<% } else if (fields[fieldId].fieldType == 'Integer') { %>
+    <%= fields[fieldId].fieldName %> varchar(<%=fields[fieldId].fieldLength%>) <% if (fields[fieldId].fieldRequired) { %>NOT NULL<% } %>,<% } else if (fields[fieldId].fieldType == 'Integer') { %>
```

Comment:
Yes this is not correct, my bad, I forgot that cassandra does not support fixed length varchars.